roles/azure_configure_archost/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+galaxy_info:
+  author: Bill Peck and Klaas Demter
+  description: Role to configure ARC host in Azure Cloud.
+  company: Red Hat
+
+  license: GPL-3.0-or-later
+
+  min_ansible_version: 2.15.0
+
+  platforms:
+    - name: EL
+      versions:
+        - "8"
+
+  galaxy_tags:
+    - linux
+    - system
+    - ansible
```

Comment:
add azure tag?

Reply:
None of the other roles have azure as a tag.  But I think it's a good idea so I will do another PR to add that tag to all roles.